chapters/1-rationale.md

```diff
@@ -70,11 +70,17 @@ In an SPDX document, Relationship elements can be used to indicate relationships
 
 **1.7.5** Must permit automated specification syntax validation.
 
-**1.7.6** Resource Description Framework (RDF) can be used to represent this information, as can an annotated tag value flat text file.
+**1.7.6** Multiple file formats can be used to represent the information being exchanged.   Current supported formats include:
+* YAML 1.2  see: https://yaml.org/spec/1.2/spec.html
+* JavaScript Object Notation (JSON) see: ECMA-404(https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf)
+* Extensible Markup Language (XML) see: https://www.w3.org/TR/2008/REC-xml-20081126/
+* Resource Description Framework (RDF also referred to as RDF/XML) see: https://www.w3.org/TR/rdf-syntax-grammar/
+* `tag:value` flat text file as described in this specification
+* .xls spreadsheets
```

Comment:
I would suggest removing .xls spreadsheets from the list. Even if the Java tools might handle them, I don't believe they're defined in the spec or included in the new `examples/` folder like the new JSON / YAML / XML formats.

Reply:
Spreadsheet format on WIKI page will need to be updated (move to github repo,  first 2.1 version,  then apply 2.2 after OWL).

Reply:
Key is to make sure we have an examples present.